skills/golem-powers/notify/SKILL.md

```
@@ -1,10 +1,10 @@
 ---
 name: notify
-description: Send Telegram notification to user when task completes
+description: Use when sending Telegram notifications after task completion, blockers, or checkpoints. Covers notify, alert, ping user.
 ---
 
 ```bash
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)